crengine/src/private/lvfreetypeface.cpp

```diff
@@ -1072,8 +1087,15 @@ bool LVFreeTypeFace::getGlyphInfo(lUInt32 code, LVFont::glyph_info_t *glyph, lCh
         // So, with such fake italic, the values we'll use below are wrong,
         // and may cause some wrong glyphs positioning or advance.
         FT_GlyphSlot_Oblique(_slot); // This uses FT_Outline_Transform(), see freetype2/src/base/ftsynth.c
-        
-        // QT has some code that seem to fix these metrics in transformBoundingBox() at
+    }
+#if 0
+    if (_synth_weight > 0 || _italic==2) {
+        // Render now that transformations are applied
+        FT_Render_Glyph(_slot, _drawMonochrome?FT_RENDER_MODE_MONO:FT_RENDER_MODE_LIGHT);
+    }
+#endif
+    if (_italic == 2) {
```

Comment:
(Some dead code that made you split the single `if (_italic == 2) {` into two parts.)

Reply:
Delete it?

Reply:
Well, depends on how clean you want your commits into CoolReader master :) Your 3rd commit has also some bits of work in progress that are removed by the 4th.
(Would have saved me a few minutes in the process of merging, but now that it's done, I don't mind :)